tools/peview/peexceptprp.c

```diff
@@ -166,7 +166,80 @@ VOID PvEnumerateExceptionEntries(
     }
     else if (imageMachine == IMAGE_FILE_MACHINE_ARM64)
     {
-        /* Todo */
+        for (ULONG i = 0; i < exceptions.NumberOfEntries; i++)
+        {
+            PPH_IMAGE_RUNTIME_FUNCTION_ENTRY_ARM64 entry = PTR_ADD_OFFSET(exceptions.ExceptionDirectory, i * sizeof(PH_IMAGE_RUNTIME_FUNCTION_ENTRY_ARM64));
```

Comment:
`IMAGE_RUNTIME_FUNCTION_ENTRY_ARM64` is in the SDK headers now, I think we can eliminate the `PPH_IMAGE_RUNTIME_FUNCTION_ENTRY_ARM64`?